packages/migrate/src/MigrateEngine.ts

```diff
@@ -51,78 +52,171 @@ export class MigrateEngine {
     this.debug = debug
     this.enabledPreviewFeatures = enabledPreviewFeatures
   }
-  public stop(): void {
-    this.child!.kill()
-  }
+
   /* eslint-disable @typescript-eslint/no-unsafe-return */
+  //
+  // See JSON-RPC API definition:
+  // https://prisma.github.io/prisma-engines/doc/migration_core/json_rpc/index.html
+  //
 
-  // Runs dev diagnostic
-  public devDiagnostic(args: EngineArgs.DevDiagnosticInput): Promise<EngineResults.DevDiagnosticOutput> {
-    return this.runCommand(this.getRPCPayload('devDiagnostic', args))
+  /**
+   * Apply the migrations from the migrations directory to the database.
+   * This is the command behind prisma migrate deploy.
+   */
+  public applyMigrations(args: EngineArgs.ApplyMigrationsInput): Promise<EngineResults.ApplyMigrationsOutput> {
+    return this.runCommand(this.getRPCPayload('applyMigrations', args))
   }
-  // List migrations in migration directory.
-  public listMigrationDirectories(
-    args: EngineArgs.ListMigrationDirectoriesInput,
-  ): Promise<EngineResults.ListMigrationDirectoriesOutput> {
-    return this.runCommand(this.getRPCPayload('listMigrationDirectories', args))
+
+  /**
+   * Create the logical database from the Prisma schema.
+   */
+  public createDatabase(args: EngineArgs.CreateDatabaseInput): Promise<EngineResults.CreateDatabaseOutput> {
+    return this.runCommand(this.getRPCPayload('createDatabase', args))
   }
```

Comment:
This is a new method: https://prisma.github.io/prisma-engines/doc/migration_core/json_rpc/index.html#-createdatabase